### DISCUSSION TITLE
Can't make theme to work. Missing styles?

### DISCUSSION THREAD

**Discussion by xuanngo2001** [filipecarneiro/hugo-bootstrap-theme#11] [Q&A] [answered]
I did the followings with hugo v0.124.1 extended version:

```bash
hugo new site MyNewSite
cd MyNewSite

npm install hugo-installer
npm install @filipecarneiro/hugo-bootstrap-theme

```

Add in `hugo.toml`:

```
theme = "hugo-bootstrap-theme"
themesdir = "node_modules/@filipecarneiro"
```

Copied your `about.en.md` to `./content/about.en.md`.

hugo server -D 

The result is. It doesn't look like the demo site. It looks like it is missing some styles?

![image](https://github.com/filipecarneiro/hugo-bootstrap-theme/assets/8182563/2eef501f-bb68-4f14-b657-5c30b54716f3)

**Comment by filipecarneiro** [comment]
Hi Xuan,

Thank you for the feedback!

If you look at `http://localhost:1313/css/style.css`, or inspect any elements on that page, you can see that Bootstrap and theme CSS is being applied.

Add some [configuration](https://gohugo.io/getting-started/configuration/), like copyright, description and your menus.

Have a look on exampleSite for inspiration: https://github.com/filipecarneiro/hugo-bootstrap-theme/tree/main/exampleSite/config/_default

If you prefer, copy the complete exampleSite, start from there, and then change or remove what you don't want.

**Comment by xuanngo2001** [comment]
Open `http://localhost:1313/css/style.css` does show the css code.

Copy the whole exampleSite doesn't work either. Hugo stops working and shows git error, which doesn't make sense.
```bash
rm -rf MyNewSite
hugo new site MyNewSite
cd MyNewSite

npm install hugo-installer
npm install @filipecarneiro/hugo-bootstrap-theme

\cp -a node_modules/\@filipecarneiro/hugo-bootstrap-theme/exampleSite/* .
sed -i 's|themesdir = .*|themesdir = "node_modules/@filipecarneiro"|' ./config/_default/config.toml

hugo server -D 
```
```
ERROR Failed to read Git log: fatal: not a git repository (or any parent up to mount point /media)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
```

The only way to load the whole exampleSite is to go through git clone:
```bash
git clone https://github.com/filipecarneiro/hugo-bootstrap-theme.git
cd hugo-bootstrap-theme
hugo server -D --source exampleSite
```

**Comment by filipecarneiro** [comment, marked as answer]
That’s because GitInfo is enabled on that project!

Just change to `enableGitInfo = false` in config.toml or initialize git on that folder with `git init`

Note: If you are coping the example site, you should not create a new hugo site (`hugo new site`)!

**Reply by filipecarneiro** [reply]
See https://gohugo.io/getting-started/configuration/#enablegitinfo

**Comment by filipecarneiro** [comment]
I’ve commented out GitInfo from the example site. It’s useful in production but not needed in the example site.
